cmd/ctr/commands/run/run_unix.go

```diff
@@ -465,7 +464,7 @@ func getNetNSPath(_ gocontext.Context, task containerd.Task) (string, error) {
 // This is used from the `run` command to avoid creating a registry with auto-refresh enabled.
 // It also provides a way to override the CDI spec file paths if required.
 func withStaticCDIRegistry() oci.SpecOpts {
-	return func(ctx context.Context, _ oci.Client, _ *containers.Container, s *oci.Spec) error {
+	return func(ctx gocontext.Context, _ oci.Client, _ *containers.Container, s *oci.Spec) error {
```

Comment:
If we're removing the duplicate import, can we keep the standard name instead?

Reply:
@samuelkarp 
I have found that gocontext is used in other packages, and context is used as an internal variable in this file, so I choose to keep goctext. If I need to keep the original import, I can modify it

+ other package

https://github.com/containerd/containerd/blob/d297fbee39ca5ea54dabdb3b40ae4e1b45954e79/cmd/ctr/commands/client.go#L20

+ this package use context in function

```go
https://github.com/containerd/containerd/blob/d297fbee39ca5ea54dabdb3b40ae4e1b45954e79/cmd/ctr/commands/run/run_unix.go#L86
```

Reply:
Agreed `context` is preferable, but think this PR is still valid as it consistent and cleans up. Would love to see a follow up that changes all the cases under `cmd` to use `context` for "context" and maybe `clicontext` for the conflicting one.

```
$ grep -rn "gocontext \"context\"" cmd
cmd/containerd/command/config.go:20:    gocontext "context"
cmd/containerd/command/main.go:20:      gocontext "context"
cmd/containerd/command/publish.go:20:   gocontext "context"
cmd/ctr/commands/images/push.go:20:     gocontext "context"
cmd/ctr/commands/run/run.go:20: gocontext "context"
cmd/ctr/commands/run/run_unix.go:23:    gocontext "context"
cmd/ctr/commands/run/run_windows.go:20: gocontext "context"
cmd/ctr/commands/shim/io_unix.go:22:    gocontext "context"
cmd/ctr/commands/shim/shim.go:22:       gocontext "context"
cmd/ctr/commands/snapshots/snapshots.go:20:     gocontext "context"
cmd/ctr/commands/tasks/tasks.go:20:     gocontext "context"
cmd/ctr/commands/tasks/delete.go:20:    gocontext "context"
cmd/ctr/commands/tasks/tasks_unix.go:22:        gocontext "context"
cmd/ctr/commands/tasks/tasks_windows.go:20:     gocontext "context"
cmd/ctr/commands/client.go:20:  gocontext "context"
cmd/ctr/commands/resolver.go:21:        gocontext "context"
cmd/ctr/commands/signals.go:20: gocontext "context"

```